src/compas_fab/robots/robot.py

```diff
@@ -1097,36 +1220,36 @@ def plan_cartesian_motion(self, frames_WCF, start_configuration=None,
                               avoid_collisions=True, group=None,
                               path_constraints=None,
                               attached_collision_meshes=None):
-        """Calculates a cartesian motion path (linear in tool space).
+        """Calculate a cartesian motion path (linear path in tool space).
 
         Parameters
         ----------
-        frames_WCF: list of :class:`compas.geometry.Frame`
+        frames_WCF : :class:`list` of :class:`compas.geometry.Frame`
             The frames through which the path is defined.
-        start_configuration: :class:`Configuration`, optional
+        start_configuration : :class:`Configuration`, optional
             The robot's full configuration, i.e. values for all configurable
             joints of the entire robot, at the starting position. Defaults to
             the all-zero configuration.
-        max_step: float
+        max_step : :class:`float`, optional
             The approximate distance between the calculated points. (Defined in
             the robot's units)
-        jump_threshold: float
+        jump_threshold : :class:`float`, optional
             The maximum allowed distance of joint positions between consecutive
             points. If the distance is found to be above this threshold, the
             path computation fails. It must be specified in relation to max_step.
             If this theshhold is 0, 'jumps' might occur, resulting in an invalid
-            cartesian path. Defaults to pi/2.
-        avoid_collisions: bool, optional
-            Whether or not to avoid collisions. Defaults to True.
-        group: str, optional
+            cartesian path. Defaults to pi / 2.
```

Comment:
Perhaps ":math:`\pi/2`" ?

Reply:
I want to avoid making this a raw docstring since that would break the doctest (since it contains \). I’ll change it to ”:math:\\pi/2”.

Reply:
oh, right, well don't break the doctest...

Reply:
8616817